models/resnet.py

```diff
@@ -54,9 +55,10 @@ def __call__(self, x):
     return out
 
 class ResNet:
-  def __init__(self, num, num_classes, groups=1, width_per_group=64):
+  def __init__(self, num, num_classes=None, groups=1, width_per_group=64, stride_in_1x1=False):
     self.num = num
-
+    self.num_classes = num_classes
```

Comment:
Neither of these should be on the class.

Reply:
removed